example/android/app/src/main/AndroidManifest.xml

```diff
@@ -35,5 +35,24 @@
                 <category android:name="android.intent.category.LAUNCHER"/>
             </intent-filter>
         </activity>
+        <activity
+            android:name=".EmbeddingV1Activity"
```

Comment:
is this for some sort of backwards compatibility? because my understanding is this PR is migrating from V1 -> V2, I'm not totally familiar with the requirements but I expect things to be V2 now (removing V1 code).